modules/social_features/social_profile/social_profile.install

```diff
@@ -229,3 +229,126 @@ function social_profile_update_130002(): void {
     ->save();
   \Drupal::logger('social_profile')->info('The view "newest_users" has been updated with new configuration.');
 }
+
+/**
+ * Create entity-form-display or hidden path field from Profile Tag taxonomy.
+ */
+function social_profile_update_130003(): void {
```

Comment:
function `social_profile_update_130003` and `social_profile_update_130004` and `social_profile_update_130005` are very duplicated code to me.
Can we create a service to use instead of copy pasting?

Reply:
Ideally `social_content_report_update_130002` would be using the service as well if social_profile and social_content_report share a module dependency.

Reply:
I created a function to avoid duplicate hook-update with same code, but I done only for Social Profile, because create a Service would complex because in Cablecar isn't be possible to use before a release.

These hook-update will be deleted in a near future, but let me know if you have other idea.